src/elements/content-sidebar/Sidebar.js

```diff
@@ -48,6 +50,7 @@ type Props = {
     getViewer: Function,
     hasActivityFeed: boolean,
     hasAdditionalTabs: boolean,
+    hasBoxAI: boolean,
```

Comment:
❓ is this prop actually used? Seems like `{isFeatureEnabledInContext(this.props.features, 'boxai.sidebar.enabled')}` is passed above but nothing consumes `this.props.hasBoxAI`. The on/off value is re-derived from `canHaveBoxAISidebar`